src/js/edu-benefits/5490/config/form.js

```diff
@@ -67,7 +85,140 @@ const formConfig = {
     },
     militaryService: {
       title: 'Military History',
-      pages: {}
+      pages: {
+        sponsorVeteran: {
+          title: 'Sponsor Veteran',
+          path: 'military-service/sponsor-veteran',
+          uiSchema: {
+            relationship: {
+              'ui:title': 'What is your relationship to the Servicemember whose benefit is being transferred to you?',
+              'ui:widget': 'radio'
+            },
+            spouseInfo: {
+              divorcePending: {
+                'ui:title': 'Is there a divorce or annulment pending to the qualifying individual?',
+                'ui:widget': 'yesNo'
+              },
+              remarried: {
+                'ui:title': 'If you are the surviving spouse, have you remarried?',
+                'ui:widget': 'yesNo'
+              },
+              remarriageDate: _.assign(date.uiSchema('Remarriage Date'), {
+                'ui:options': {
+                  // Needs the || because it's undefined on other pages
+                  hideIf: (fieldData) => !fieldData.spouseInfo || !fieldData.spouseInfo.remarried
+                }
+              }),
+              'ui:options': {
+                hideIf: (fieldData) => fieldData.relationship !== 'spouse'
+              }
+            },
+            relativeFullName: _.assign(fullName.uiSchema, {
+              'ui:title': 'Name of Sponsor'
+            }),
+            // This isn't running validation for some reason...
+            veteranSocialSecurityNumber: ssn.uiSchema,
+            veteranDateOfBirth: date.uiSchema('Date of Birth'),
+            veteranDateOfDeath: date.uiSchema('Date listed as MIA, POW, or as deceased'),
+          },
+          schema: {
+            type: 'object',
+            definitions: {
+              'ui:title': 'Sponsor Service',
```

Comment:
No, I mean I'm not sure why it's in the definitions object in the regular schema. I think it should go in uiSchema somewhere.